metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.ts

```diff
@@ -22,33 +22,35 @@ import { PcapRequest } from '../model/pcap.request'
 import { Pdml } from '../model/pdml'
 import {Subscription} from "rxjs/Rx";
 
+class Query {
```

Comment:
What is the purpose of this?

Reply:
Thanks Ryan! It's dead code left there from some previous state of impl. Removed.